pkg/alerts/nrql_conditions_integration_test.go

```diff
@@ -4,24 +4,27 @@ package alerts
 
 import (
 	"fmt"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+	"github.com/stretchr/testify/require"
 	"strconv"
 	"testing"
 
-	"github.com/stretchr/testify/require"
-
-	"github.com/newrelic/newrelic-client-go/pkg/errors"
 	mock "github.com/newrelic/newrelic-client-go/pkg/testhelpers"
 )
 
 var (
 	testNrqlConditionRandomString       = mock.RandSeq(5)
-	nrqlConditionBaseThreshold          = 1.0          // needed for setting pointer
-	nrqlConditionBaseThresholdZeroValue = float64(0)   // needed for setting pointer
-	nrqlConditionBaseSignalFillValue    = float64(0.1) // needed for setting pointer
-	nrqlConditionBaseExpirationDuration = 1200         // needed for setting pointer
-	nrqlConditionBaseEvalOffset         = 3            // needed for setting pointer
-	nrqlConditionBaseAggWindow          = 60           // needed for setting pointer
-	nrqlConditionBase                   = NrqlConditionBase{
+	nrqlConditionBaseThreshold          = 1.0                                  // needed for setting pointer
+	nrqlConditionBaseThresholdZeroValue = float64(0)                           // needed for setting pointer
+	nrqlConditionBaseSignalFillValue    = float64(0.1)                         // needed for setting pointer
+	nrqlConditionBaseExpirationDuration = 1200                                 // needed for setting pointer
+	nrqlConditionBaseEvalOffset         = 3                                    // needed for setting pointer
+	nrqlConditionBaseAggWindow          = 60                                   // needed for setting pointer
+	nrqlConditionBaseAggMethod          = AlertsAggregationMethodTypes.CADENCE // needed for setting pointer
+	nrqlConditionBaseAggDelay           = 2                                    // needed for setting pointer
+	nrqlConditionBaseAggTimer           = 5                                    // needed for setting pointer
+
+	nrqlConditionCreateBase = NrqlConditionCreateBase{
```

Comment:
Since I chose to create new Create and Update structs to allow the new Streaming Methods to be nulled out when a condition is updated, this required updating the existing tests to use the correct types.